sig-apps/agenda.md

```diff
@@ -4,7 +4,7 @@ _Note, the [minutes and agenda have moved to Google Docs](https://docs.google.co
 
 ## August 17, 2016
 * Intro / Agenda
-* Brian Hardock of the Deis Helm team will demo [Helm](www.github.com/kubernetes/helm).
+* Brian Hardock of the Deis Helm team will demo [Helm](https://github.com/kubernetes/helm).
```

Comment:
Since Helm moved to its own org, the kubernetes/helm link redirects to the helm/helm repo. Would be good to update this link directly to https://github.com/helm/helm.